app/src/main/java/io/apicurio/registry/events/EventSourcedRegistryStorage.java

```diff
@@ -192,6 +193,12 @@ public void updateArtifactMetaData(String groupId, String artifactId, EditableAr
         //no event here, UPDATE_ARTIFACT is for cases where a new version is added
     }
 
+    @Override
+    public void updateArtifactOwner(String groupId, String artifactId, ArtifactOwnerDto owner) throws ArtifactNotFoundException, RegistryStorageException {
+        super.updateArtifactOwner(groupId, artifactId, owner);
+        //TODO consider a change ownership event
```

Comment:
Maybe the delegate should be used here instead of super? btw, I agree, adding an event here makes sense :)

Reply:
It probably makes sense to just remove any of the methods that do nothing but call the delegate, since the parent class is already doing that.  Usually when you extend a class in this way that's what you do, so that the subclass only overrides the functionality it needs.  But I've made the change you suggested instead because I don't feel very strongly about it and using the delegate directly is what we're doing everywhere else.  So +1 for consistency.